resources/windows/windows.pkr.hcl

```diff
@@ -66,8 +101,9 @@ build {
     for_each = local.win_factory.images
     labels   = ["amazon-ebs.win-source"]
     content {
-      ami_name = source.value.name
-      name     = source.value.name
+      # Specify the the AMI name, i.e. metasploit-windows-builder-1.0.8
+      ami_name = "${source.value.name}-${source.value.version}"
```

Comment:
This allows us to build new versions of the AMI without replacing the existing working versions